vscode/polar-lang/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "oso",
+  "publisher": "osohq",
+  "displayName": "oso",
+  "description": "Syntax highlighting for oso .polar files",
+  "icon": "oso_logo.png",
+  "version": "0.5.0",
+  "engines": {
+    "vscode": "^1.43.0"
+  },
+  "categories": [
+    "Programming Languages"
+  ],
+  "contributes": {
+    "languages": [
+      {
+        "id": "oso",
+        "aliases": [
+          "Oso",
+          "oso"
+        ],
+        "extensions": [
+          ".polar"
+        ],
+        "configuration": "./language-configuration.json"
+      }
```

Comment:
should the language be "Polar"? This seems like a confusing one. If I'm editing a .polar file, should the little bottom bar in VS Code say "Polar" or "oso"? It is an oso extension that (among other future things) provides syntax highlighting for the Polar language.

Reply:
Ah good point. Polar would be fine here.